frontend/src/Episode/History/EpisodeHistoryRow.js

```diff
@@ -102,14 +101,28 @@ class EpisodeHistoryRow extends Component {
           />
         </TableRowCell>
 
+        <TableRowCell>
+          <EpisodeFormats formats={customFormats} />
+        </TableRowCell>
+
+        <TableRowCell>
+          <Tooltip
+            anchor={
+              formatCustomFormatScore(customFormatScore, customFormats.length)
+            }
+            tooltip={<EpisodeFormats formats={customFormats} />}
+            position={tooltipPositions.BOTTOM}
+          />
+        </TableRowCell>
+
         <RelativeDateCellConnector
           date={date}
         />
 
-        <TableRowCell className={styles.details}>
+        <TableRowCell className={styles.actions}>
           <Popover
             anchor={
-              <Icon
+              <IconButton
```

Comment:
Why is it changing to a button?

Reply:
I understand that grouping these saves horizontal space, but information under `Actions` doesn't seem right and we should consider removing the column title.